dalgo-kubernetes/superset-kubernetes/staging/demosuperset/README.md

```diff
@@ -5,7 +5,7 @@ demosuperset
 kubectl apply -f superset-env.yaml
 
 # install/upgrade
-helm upgrade --install --values values4.1.1.yaml  demosuperset superset/superset --version 0.14.1 --namespace superset --debug
+helm upgrade --install --values values5.0.0.yaml  demosuperset superset/superset --version 0.14.1 --namespace superset --debug
```

Comment:
do you want to just keep the older command also, just for the history. we can comment it out but keep.

upto to you. but looking good.